api/controllers/TelevisionController.js

```diff
@@ -0,0 +1,319 @@
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/:id/mute Set muted 
+     * @apiName televisionSetMuted
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * @apiParam {boolean} muted true if sound is muted
+     */
+    setMuted: function(req, res, next){
+        gladys.television.setMuted(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+     /**
+     * @api {post} /television/volume Set Volume 
+     * @apiName televisionSetVolume
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * @apiParam {integer} volume Volume level
+     */
+    setVolume: function(req, res, next){
+        gladys.television.setVolume(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/stop Stop 
+     * @apiName televisionStop
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    stop: function(req, res, next){
+        gladys.television.stop(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/volume/up Volume Up 
+     * @apiName televisionVolumeUp
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    volumeUp: function(req, res, next){
+        gladys.television.volumeUp(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/volume/down Volume Down 
+     * @apiName televisionVolumeDown
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    volumeDown: function(req, res, next){
+        gladys.television.volumeDown(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/presskey Press Key 
+     * @apiName televisionPressKey
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    pressKey: function(req, res, next){
+        gladys.television.pressKey(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {get} /television/source Get Sources
+     * @apiName televisionGetSources
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * 
+     * @apiSuccess {array} All television source
+     */
+    getSources: function(req, res, next){
+        gladys.television.getSources(req.query)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/opensources Open Sources
```

Comment:
POST `/television/opensources`

Please keep the route singular! 

(I know every routes in Gladys are not singular, but most are)

Reply:
Fix this on commit 3ae5f94